### PR TITLE
Update solution name due to rebranding

### PR DESCRIPTION
**Pull request by rikukawano**

### CODE REVIEW

**Comment by raimundo-henriques** [comment]
@rikukawano Thank you for the update! 

As your forked repo was based on an old version of the catalog, your `solution.json` still had the `"provider"` field, which was meanwhile replaced by the `"provider_id"` field. The goal of this change was to make it clearer that the value should be the `user-id` of the provider and not their name. 

I took the liberty to commit the necessary changes, I hope that is fine. Can you please confirm that you've seen and agree with them by answering this commentary? 

Thank you!
CC @prasadt1

File: catalog/solutions/zeroboard/2.20.0/solution.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
   "name": "Zeroboard",
   "version": "2.20.0",
   "website": "https://zeroboard.jp",
-  "provider": "Zeroboard",
+  "provider_id": "zeroboard",
```
Comment (prasadt1) [inline catalog/solutions/zeroboard/2.20.0/solution.json:6]: @raimundo-henriques - Should the vale be "Zeroboard" instead of zeroboard?

Comment (raimundo-henriques) [reply catalog/solutions/zeroboard/2.20.0/solution.json:6]: Here it should actually be `"zeroboard"`, as the value of `"provider_id"` should be the `user-id` (that only admits lowercase letters) and not the provider name

Comment (prasadt1) [reply catalog/solutions/zeroboard/2.20.0/solution.json:6]: @raimundo-henriques - Thanks! It's clear now.

**Comment by rikukawano** [comment]
@raimundo-henriques 
Thank you for making the necessary changes! It all looks good to me.

**Review by prasadt1 — APPROVED** [review]

**Review by prasadt1 — APPROVED** [review]

**Pull request merged**